parquet-hadoop/src/main/java/org/apache/parquet/format/converter/ParquetMetadataConverter.java

```diff
@@ -534,9 +584,8 @@ private void addRowGroup(
     int columnOrdinal = -1;
     ByteArrayOutputStream tempOutStream = null;
     for (ColumnChunkMetaData columnMetaData : columns) {
-      // There is no ColumnMetaData written after the chunk data, so set the ColumnChunk
-      // file_offset to 0
-      ColumnChunk columnChunk = new ColumnChunk(0);
+      ColumnChunk columnChunk =
```

Comment:
It seems that you have not rebased on the latest master branch. Please revert these two lines.